package.json

```diff
@@ -24,9 +24,12 @@
   "author": "Scott Newcomer",
   "license": "MIT",
   "dependencies": {
+    "@embroider/macros": "^1.8.3",
     "ember-auto-import": "^2.2.3",
     "ember-cli-babel": "^7.26.6",
-    "ember-modifier": "^2.1.2 || ^3.0.0",
+    "ember-destroyable-polyfill": "^2.0.3",
+    "ember-modifier": "^3.2.7",
+    "ember-modifier-manager-polyfill": "1.2.0",
```

Comment:
@And012 we should not introduce `ember-modifier-manager-polyfill` as it's needed for Ember versions 3.8 and below and this addon is claim to be compatible with 3.16 and up